app/Observers/UserObserver.php

```diff
@@ -43,6 +43,19 @@ public function saving(User $user)
             }
         }
 
+        //Purify Billing info
+        if (!empty($user->profile['billing'])) {
+            $profile = $user->profile;
+            $profile['billing'] = mb_substr(strip_tags($profile['billing']), 0, 1024);
```

Comment:
Replace strip_tags with the "purify" system we use to avoid people writing javascript and other nonsense